app/assets/stylesheets/retro.scss

```diff
@@ -1,41 +1,34 @@
-#nav {
+.nav {
   background-color: #696969;
   height: 60px;
+}
 
-  .headerIcons {
-    text-align: right;
+.headerItems {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+}
 
-    .header-icon {
-      border-radius: 30px;
-      background-color: white;
-      padding: 4px;
-      margin-top: 7px;
-      margin-right: 15px;
-      vertical-align: middle;
-    }
+.header-icon {
+  border-radius: 30px;
+  background-color: white;
+  padding: 4px;
+  margin-right: 15px;
+  margin-top: 6px;
+}
 
-    #time {
-      outline: none;
-      border-top-left-radius: 10px;
-      border-top-right-radius: 10px;
-      font-weight: bold;
-      font-size: 17px;
-      color: #02a397;
-      display: none;
-      vertical-align: middle;
-      margin-right: 44%;
-    }
+#time {
+  outline: none;
+  border-top-left-radius: 10px;
```

Comment:
Just to keep in mind, `border-radius` is a [shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). This means that do something like
`border-radius: 1px 2px 3px 4px;` 
instead of
```
border-top-left-radius: 1px;
border-top-right-radius: 2px;
border-bottom-left-radius: 3px;
border-bottom-right-radius: 4px;
``` 
or 
`border-radius: 10px 10px 0px;`
instead of
```
border-top-left-radius: 10px; 
border-top-right-radius: 10px; 
```
There are lots of shorthand properties, some of them are `margin, padding, border-radius, border, background, etc`. This can be really helpful when you want to specify more than one property, or if you want to apply the same property with different values depending on the side 👍